src/functionalTest/resources/features/F-000/F-000.feature

```diff
@@ -19,20 +19,3 @@ Feature:F-000 Access Organisation Role Mapping API
     Given a user with [an active caseworker profile],
 
 
-  @S-111
-  @FeatureToggle(DB:iac_1_0=on)
-  Scenario: must access Role Assignment API
-    Given a user with [an active caseworker profile],
-
-
-  @S-112
-  @FeatureToggle(DB:iac_1_0=on) @FeatureToggle(DB:iac_1_1=off)
-  Scenario: must access Role Assignment API
-    Given a user with [an active caseworker profile],
-
-
-  @S-113
-  @FeatureToggle(DB:iac_1_0=on) @FeatureToggle(DB:iac_1_1=on) @FeatureToggle(LD:get-ld-flag=on)
-  Scenario: must access Role Assignment API
-    Given a user with [an active caseworker profile],
-
```

Comment:
we shouldn't remove these scenarios instead we can just remove the iac_1_0 flags. The purpose of these dummy scenario is to check the DB and LD flags check mechanism. If these are removed we don't know whether this mechanism is working as expected or not.